tests/Fixer/Alias/MbStrFunctionsFixerTest.php

```diff
@@ -62,5 +62,18 @@ public function strtolower($a);
                 }',
             ],
         ];
+
+        if (\function_exists('mb_str_split')) {
```

Comment:
Tests shall be as much static as possible: regardless of the ongoing topic about how to fix this in the Fixer, the test should check the PHP version instead of existing functionality. I repeat: the test should do this, not what the test tests. /cc @SpacePossum

Reply:
the PHP version check sounds better to me, but I'm pragmatic with unit tests so not a blocker for me either